spec/requests/public_recipes_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'rails_helper'
+require_relative 'test_helper'
+
+RSpec.describe 'Recipes Index Page', type: :system do
+  add_info_to_db
+
+  scenario 'User can see the list of recipes' do
+    login(user.email, user.password)
+    visit '/recipes_public'
+    sleep(1) # Wait for the page to load
+
+    expect(page).to have_content('Public Recipes')
+    expect(page).to have_content('Test Recipe')
```

Comment:
- [ ] a test is failing, kindly fix it
![image](https://github.com/gdumani/recipify/assets/66097371/60164ecd-5ec8-4231-a231-383f52340638)

Reply:
Done! Thank you